e3sm_diags/plot/arm_diags_plot.py

```diff
@@ -0,0 +1,647 @@
+import math
+import os
+import warnings
+
+import matplotlib
+import numpy as np
+from matplotlib.gridspec import GridSpec
+
+from e3sm_diags.driver.utils.diurnal_cycle import fastAllGridFT
```

Comment:
@tomvothecoder I found a reference to the old diurnal_cycle function. I made changes locally but didn't push. 
Could you update the line to use following instead:
from e3sm_diags.driver.utils.diurnal_cycle_xr import _fft_all_grid

Reply:
Done